package/gem2rpm.yml

```diff
@@ -47,7 +45,7 @@
 #   install -D -m 0644 %{S:1} %{buildroot}%{_bindir}/gem2rpm-opensuse
 # ## used by gem2rpm
 :testsuite_command: |-
-   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake check:ci)
+   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake --verbose --trace check:ci)
```

Comment:
`(pushd foo; bar)` is superfluous needless extra duplication ;-) Use `pushd foo; bar; popd` or `(cd foo; bar)`.